casanovo/config.yaml

```diff
@@ -15,7 +15,8 @@ precursor_mass_tol: 50  # ppm
 isotope_error_range: [0, 1]
 # The minimum length of predicted peptides.
 min_peptide_len: 6
-# Number of spectra in one inference batch.
+# Number of spectra in one inference batch. 
+# Also the number of PSMs processed by Casanovo-DB per batch.
```

Comment:
nit: I think this just makes the config file more confusing

Reply:
will fix